### DISCUSSION TITLE
Alternatives to Centos 8

### DISCUSSION THREAD

**Discussion by dvdsnchzIT** [bluebanquise/bluebanquise#510] [General] [answered]
Hi,

I open a thread because I'm actually using Centos 7 (rockscluster) for my cluster.

But in the future I will have to upgrade and rebuild it.

Rockscluster is in my opinion dead with SDSC switch to Bright, so I'm looking alternatives, like openhpc or bluebanquise.

At the begining I wanted to go on Centos 8, but with IBM last decisions to kill it for Centos Stream, I'm thinking about all of this.

Will you take a look at RHEL new forks like Rocky Linux ? Or adding Ubuntu LTS/Debian as fully supported distros ?

Regards,

David

**Comment by oxedions** [comment, marked as answer]
Dear @dvdsnchzIT 

My apologies for the late answer. This end of year has been kind of "busy" for me, since my second child just born beginning of December, and so I am a bit "disconnected" with 2-3 hours sleep nights 😊  (be back online in January).

Regarding CentOS 8, yes, this is a big concern for me either, as our own Algoric cluster is on Centos 8.

To answer your question, next BlueBanquise version, 1.5, will focus on integrating support for the following distributions:

* RHEL
* CentOS Stream
* Rocky Linux + Oracle Linux (both Centos 8 equivalent)

* Debian
* Ubuntu (LTS only, so 20.04)

* Open Suse Leap

Note that only the core of the stack will be compatible with all these distributions, add-ons will depend of good will of developers (some add-ons will be easily upgraded, others may require more efforts).

I will release 1.4 in January, then start all these devs. OpenSuse should be relatively quick to implement, same for CentOS Stream and Rocky Linux/Oracle Linux. Debian and Ubuntu will take longer (issues with PXE, Debian fail to see disks, Ubuntu 20.04 introduce a new auto install mechanism). I hope all to be done before end of march.

I can focus first on a specific distribution if you need it.

I wish you a merry Christmas ! 🎄 

Ox

**Comment by dvdsnchzIT** [comment]
No problem Ox, thanks for your reply !

And best wishes for 2021 :-)
